### DISCUSSION TITLE
NextUi conflict with tailwind css

### DISCUSSION THREAD

**Discussion by IbrahimYahyaoui** [heroui-inc/heroui#1114] [Help]
When I added NextUIProvider, it caused a conflict with Tailwind CSS, resulting in a disorganized appearance of the website's CSS. I attempted to resolve the issue by adding !important or prefix to Tailwind's configuration, but these solutions did not work. Are there any other solutions to this problem? I use vite btw

**Comment by enaluz** [comment]
Bumping this ^
